### PR TITLE
Fix command descriptions and alignment

### PR DESCRIPTION
**Pull request by smarterclayton**
@liggitt / @deads2k

We need to enforce these guidelines during reviews from now on

### CODE REVIEW

File: pkg/cmd/cli/cmd/buildlogs.go [hunk 1]
```diff
@@ -17,11 +17,14 @@ import (
 )
 
 const (
-	buildLogsLong = `Retrieve logs from the containers where the build occurred.
+	buildLogsLong = `
+Retrieve logs for a build
 
-NOTE: This command may be moved in the future.`
```
Comment (deads2k) [inline pkg/cmd/cli/cmd/buildlogs.go]: Unrelated to this, but are we going to make an "oc build <subcommand>" branch to eventually retire these top level pieces (start and cancel come to mine)?

File: pkg/cmd/admin/admin.go [hunk 2]
```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
-const adminLong = `OpenShift Administrative Commands
+const adminLong = `
+OpenShift Administrative Commands
```
Comment (deads2k) [inline pkg/cmd/admin/admin.go]: Why add the newline that's just going to get trimmed later on?  Also, why force someone making a template to know that they need to trim?

File: pkg/cmd/cli/cmd/buildlogs.go (hunk 1 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/buildlogs.go]: +1 to `oc build`

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/buildlogs.go]: I don't know.  Maybe?  Maybe not?

On Mon, Jul 13, 2015 at 7:40 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/buildlogs.go
> <https://github.com/openshift/origin/pull/3669#discussion_r34454781>:
>
> >
> > -NOTE: This command may be moved in the future.`
>
> Unrelated to this, but are we going to make an "oc build " branch to
> eventually retire these top level pieces (start and cancel come to mine)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3669/files#r34454781>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

File: pkg/cmd/admin/admin.go (hunk 2 above)
Comment (smarterclayton) [inline pkg/cmd/admin/admin.go]: Commander trims automatically.  This is just making our use consistent -
the constants are more readable this way.

On Mon, Jul 13, 2015 at 7:43 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/admin/admin.go
> <https://github.com/openshift/origin/pull/3669#discussion_r34454942>:
>
> > @@ -22,7 +22,8 @@ import (
> >  	"github.com/openshift/origin/pkg/version"
> >  )
> >
> > -const adminLong = `OpenShift Administrative Commands
> > +const adminLong = `
> > +OpenShift Administrative Commands
>
> Why add the newline that's just going to get trimmed later on? Also, why
> force someone making a template to know that they need to trim?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3669/files#r34454942>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

**Comment by deads2k** [comment]
travis is failing on build and you probably need to update generated files.

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**